### PR TITLE
Brief introduction for the air quality tutorial

### PR DESCRIPTION
**Pull request by Mukulikaa**
Comments are welcome!

### CODE REVIEW

File: links-to-click/tutorial-air-quality.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-**Task**
+**Analyzing the impact of the lockdown on air quality**
 
-Provide an intro to the tutorial *Analyzing the impact of the lockdown on air quality in Delhi, India*:
-https://numpy.org/numpy-tutorials/content/tutorial-air-quality-analysis.html
+The pandemic-induced lockdown provided a rare controlled environment to observe the effect of human activity on the air quality. [Mukulika's](https://github.com/Mukulikaa) tutorial- *Analyzing the impact of the lockdown on air quality in Delhi, India*, calculates the Air Quality Indices and uses hypothesis testing on the data samples to test the improvement of air quality after the lockdown.
```
Comment (rossbar) [inline links-to-click/tutorial-air-quality.md:3]:
```suggestion
The pandemic-induced lock down provided a rare controlled environment to observe the effect of human activity on the air quality. [Mukulika's](https://github.com/Mukulikaa) tutorial- *Analyzing the impact of the lockdown on air quality in Delhi, India*, calculates the Air Quality Indices and uses hypothesis testing on the data samples to test the improvement of air quality after the lockdown.
```

File: links-to-click/tutorial-air-quality.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+
+[Check out the tutorial!](https://numpy.org/numpy-tutorials/content/tutorial-air-quality-analysis.html)
```
Comment (jakevandervaate) [inline links-to-click/tutorial-air-quality.md]:
```suggestion
[Read the the tutorial on numpy.org](https://numpy.org/numpy-tutorials/content/tutorial-air-quality-analysis.html)

[Watch a video about the tutorial on YouTube](https://www.youtube.com/watch?v=5bfFI2WuuMA)
```

File: links-to-click/tutorial-air-quality.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-**Task**
+**Analyzing the impact of the lockdown on air quality**
 
-Provide an intro to the tutorial *Analyzing the impact of the lockdown on air quality in Delhi, India*:
-https://numpy.org/numpy-tutorials/content/tutorial-air-quality-analysis.html
+The pandemic-induced lockdown provided a rare controlled environment to observe the effect of human activity on the air quality. [Mukulika's](https://github.com/Mukulikaa) tutorial- *Analyzing the impact of the lockdown on air quality in Delhi, India*, calculates the Air Quality Indices and uses hypothesis testing on the data samples to test the improvement of air quality after the lockdown.
+
```
Comment (jakevandervaate) [inline links-to-click/tutorial-air-quality.md:4]:
```suggestion

This tutorial teaches the following concepts:

* The concept of moving averages
* How to calculate Air Quality Index (AQI)
* How to perform a paired Student’s t-test and find the t and p values
* How to interpret these values

```

File: links-to-click/tutorial-air-quality.md (hunk 1 above)
Comment (jakevandervaate) [inline links-to-click/tutorial-air-quality.md]:
```suggestion
The pandemic-induced lockdown provided a rare, controlled environment to observe the effect of human activity on Delhi's air quality. [Mukulika's](https://github.com/Mukulikaa) tutorial, *Analyzing the impact of the lockdown on air quality in Delhi, India*, calculates Air Quality Indices and tests the air-quality improvement after the lockdown.
```

**Comment by Mukulikaa** [comment]
Thanks a lot, @jakevandervaate!

File: links-to-click/tutorial-air-quality.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+The pandemic-induced lockdown provided a rare, controlled environment to observe the effect of human activity on Delhi's air quality. [Mukulika's](https://github.com/Mukulikaa) tutorial, *Analyzing the impact of the lockdown on air quality in Delhi, India*, calculates Air Quality Indices and tests the air-quality improvement after the lockdown.
+
+This tutorial teaches the following concepts:
+
+* The concept of moving averages
+* How to calculate Air Quality Index (AQI)
+* How to perform a paired Student’s t-test and find the t and p values
+* How to interpret these values
+
+[Read the the tutorial on numpy.org](https://numpy.org/numpy-tutorials/content/tutorial-air-quality-analysis.html)
```
Comment (InessaPawson) [inline links-to-click/tutorial-air-quality.md]:
```suggestion
[Read a text version of the tutorial on numpy.org.](https://numpy.org/numpy-tutorials/content/tutorial-air-quality-analysis.html)
```

File: links-to-click/tutorial-air-quality.md [hunk 5]
```diff
@@ -1,4 +1,14 @@
...
+
+[Watch a video about the tutorial on YouTube](https://www.youtube.com/watch?v=5bfFI2WuuMA)
```
Comment (InessaPawson) [inline links-to-click/tutorial-air-quality.md]:
```suggestion
[Watch a video presentation of the tutorial on the official NumPy YouTube channel.](https://www.youtube.com/watch?v=5bfFI2WuuMA)
```

**Review by InessaPawson — APPROVED** [review]

**Pull request merged**